mirrord/config/src/lib.rs

```diff
@@ -323,7 +323,11 @@ impl LayerConfig {
     /// We don't call it from `from_env` since we want to verify it only once (from cli)
     ///
     /// Fills `context` with the warnings.
-    pub fn verify(&self, context: &mut ConfigContext) -> Result<(), ConfigError> {
+    ///
+    /// - `ide`: Identifies if this is being called from an IDE context, when using
+    /// `mirrord verify-config`. Turns some _target missing_ errors into warnings, as the target can
+    /// be selected after `verify-config` is run.
+    pub fn verify(&self, context: &mut ConfigContext, ide: bool) -> Result<(), ConfigError> {
```

Comment:
maybe it's better to have it under `ConfigContext`?

Reply:
You meant just the `ide` arg, right?

Reply:
yes